sqlanalytics/resource_sql_global_config.go

```diff
@@ -0,0 +1,117 @@
+package sqlanalytics
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type SQLEndpointConfPair struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+// SQLEndpointGlobalConfig ...
+type SQLEndpointGlobalConfig struct {
```

Comment:
probably remove `SQLEndpoint` prefix from all types in this file - names make lines too long for something this small :) it's `sql` package anyway, no name clashes.